ext/xsl/tests/transformToDoc_sxe_type_error.phpt

```diff
@@ -0,0 +1,52 @@
+--TEST--
+XSLTProcessor::transformToDoc return value type error with SimpleXML
+--EXTENSIONS--
+xsl
```

Comment:
simplexml here too?

Reply:
Yes thanks for spotting, I'll add it in the merge